connection.c

```diff
@@ -62,10 +62,9 @@ Connection *ConnCreate(RedisRaftCtx *rr, void *privdata, ConnectionCallbackFunc
     conn->idle_callback = idle_cb;
     conn->free_callback = free_cb;
     conn->id = ++id;
-    if (rr->config->connection_timeout > 0) {
-        conn->timeout = RedisModule_Calloc(1, sizeof(struct timeval));
-        conn->timeout->tv_usec = rr->config->connection_timeout;
-    }
+
+    conn->timeout = RedisModule_Calloc(1, sizeof(struct timeval));
```

Comment:
@sjpotter Why not keep the `struct timeval` part of the connection structure?

Reply:
oh, as in not allocated?